test/e2e/framework.go

```diff
@@ -1392,7 +1394,48 @@ func (data *TestData) deleteClickHouseOperator(chOperatorYML string) error {
 	return nil
 }
 
+func (data *TestData) killProcesses(namespace, podName, containerName, processName string) error {
+	cmds := []string{"pgrep", "-f", processName}
+	stdout, stderr, err := data.RunCommandFromPod(namespace, podName, containerName, cmds)
+	if err != nil {
+		if strings.Contains(err.Error(), "does not have a host assigned") {
+			return nil
+		}
+		return fmt.Errorf("error when getting pid of '%s', stderr: <%v>, err: <%v>", processName, stderr, err)
+	}
+	cmds = []string{"kill", "-SIGINT", strings.TrimSpace(stdout)}
+	_, stderr, err = data.RunCommandFromPod(namespace, podName, containerName, cmds)
+	if err != nil {
+		return fmt.Errorf("error when sending SIGINT signal to '%s', stderr: <%v>, err: <%v>", processName, stderr, err)
+	}
+	return nil
+}
+
+func (data *TestData) killProcessesOnPods() error {
+	listOptions := metav1.ListOptions{
+		LabelSelector: "",
+	}
+
+	pods, err := data.clientset.CoreV1().Pods(flowVisibilityNamespace).List(context.TODO(), listOptions)
+	if err != nil {
+		return fmt.Errorf("failed to list flow-visibility pods: %v", err)
+	}
+	for _, pod := range pods.Items {
+		podName := pod.Name
+		if strings.Contains(podName, "chi-clickhouse-clickhouse") {
+			err = data.killProcesses("flow-visibility", pod.Name, clickHouseMonitorContName, clickHouseMonitorContName)
```

Comment:
Maybe unify the names here? I think podName=pod.Name and clickHouseMonitorContName="chi-clickhouse-clickhouse"?

Reply:
the clickhouse monitor container is named clickhouse-monitor but the pod is chi-clickhouse-clickhouse, so I can leave that comparison as it is, but I'll fix the rest